src/ducks/recurrence/DebugRecurrencePage.jsx

```diff
@@ -124,17 +126,21 @@ const getColor = bundle => {
   return palette[h % (palette.length - 1)]
 }
 
-const List = ({ children }) => {
-  return <ul className="u-m-0 u-ph-1 u-pv-half">{children}</ul>
-}
-
 const CategoryNames = ({ bundle }) => {
   return (
-    <List>
+    <div>
```

Comment:
Do we need this div? Or a fragment is enough?

Reply:
It it not really need but not really a problem also. I don't know :) I keep it, it is in the debug page so it is not really a problem for the user.